tests/test_schema.py

```diff
@@ -608,6 +609,77 @@ def test_many_id_errors(self):
         assert id_err["source"]["pointer"] == "/data/1/id"
         assert id_err["detail"] == "Not a valid string."
 
+    def test_nested_fields_error(self):
+
+        min_size = 10
+
+        class ThirdLevel(ma.Schema):
+            number = fields.Int(required=True, validate=ma.validate.Range(min=min_size))
+
+        class SecondLevel(ma.Schema):
+            foo = fields.Str(required=True)
+            third = fields.Nested(ThirdLevel)
+
+        class FirstLevel(Schema):
+            class Meta:
+                type_ = "first"
+
+            id = fields.Int()
+            second = fields.Nested(SecondLevel)
+
+        schema = FirstLevel()
+        result = schema.validate(
+            {
+                "data": {
+                    "type": "first",
+                    "attributes": {"second": {"third": {"number": 5}}},
+                },
+            }
+        )
+        """
```

Comment:
Why a triple-quoted string instead of `# `?

Reply:
Actually, this whole string probably isn't necessary. It's fairly clear from `expected_errors` below what the...expected errors... are supposed to look like.

Reply:
Just to have the full picture
OK, just in case you think it's unnecessary, I'll remove this string